lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -211,7 +209,7 @@ Try {
                 if ($site_name) {
                     $install_params.SiteName = $site_name
                 }
-                $install_result = Install-ADDSDomainController -NoRebootOnCompletion -Force @install_params
+                Install-ADDSDomainController -NoRebootOnCompletion -Force @install_params
```

Comment:
You are not doing anything with the output which could pollute the stdout returned to Ansible, redirect to null with `> $null`.